### ISSUE TITLE
Brick after normal reboot

### ISSUE THREAD

**Issue by ksaanen** [kszaq/OpenELEC.tv#30]
Hi Kszaq,

I tried the experimental HD18Q image (6.95.2.2/nand) on my MXQ (S805_MXQ_V2.1_150516 written on the mainboard) and all works great except for any of the reboot options. They send the box into a bricked state (which can be fixed by flashing an Android rom and shorting the nand). Do you know of this issue?

Regards,
Kevin

**Comment by kszaq** [comment]
I've never heard of that issue before. Does powering off/reboot work in Android?

**Comment by ksaanen** [comment]
Yes it did, it was running on [MXQ HD18Q Multi ROM v2 (Custom)](http://mbox.co.za/forum/viewtopic.php?t=482) from mediaman before because I couldn't find the original firmware anywhere and this one was most compatible (only the LED stayed red all the time instead of turning blue on "power on").

**Comment by kszaq** [comment]
I'm afraid the only way to debug that woud be attaching serial console and looking at logs.

**Comment by ksaanen** [comment]
Ahh I don't have the needed hardware to do so at this moment. Maybe I'll try it out later, or flash a different firmware/bootloader before installing OpenElec again. Thanks for the quick replies, I'll keep an eye out on future releases.

Regards,
Kevin
